cds/utils.bal

```diff
@@ -0,0 +1,192 @@
+import ballerina/http;
+import ballerina/lang.regexp;
+import ballerinax/health.fhir.r4;
+import ballerinax/health.fhir.r4.parser;
+import ballerinax/health.fhir.r4.validator;
+
+public isolated function validateContext(CdsRequest cdsRequest, CdsService cdsService) returns CdsError? {
```

Comment:
Let's have bal docs for public functions

Reply:
Added